mmv1/templates/terraform/pre_create/datastream_connection_profile.go.tmpl

```diff
@@ -11,8 +11,8 @@
 	limitations under the License.
 */ -}}
 
-_, exists := d.GetOk("create_without_validation")
-if !exists {
-	url += "false"
-	d.Set("create_without_validation", false)
-}
\ No newline at end of file
+if d.Get("create_without_validation").(bool) {
+		url, err = transport_tpg.AddQueryParams(url, map[string]string{"force": "true"})
+	} else {
+		url, err = transport_tpg.AddQueryParams(url, map[string]string{"force": "false"})
+	}
```

Comment:
nit: the indentation is looking weird to me, although it should be fine as long as gofmt still works on it.